vision/cloud-client/detect/README.rst

```diff
@@ -94,12 +94,13 @@ To run this sample:
     python detect.py web-uri http://wheresgus.com/dog.JPG
     python detect.py web-geo ./resources/city.jpg
     python detect.py faces-uri gs://your-bucket/file.jpg
+    python detect_pdf.py ocr-uri gs://python-docs-samples-tests/HodgeConj.pdf     gs://BUCKET_NAME/PREFIX/
```

Comment:
This example has a bunch of extra whitespace, is that intentional?

Reply:
Ah, well the line was too long in detect.py, so I added spaces to make it easier to read, but I'll remove the spaces and regen the README to make it cleaner.